Core/ConfigWizard.cs

```diff
@@ -157,7 +157,7 @@ public static void RunConfigWizard(bool useDefault)
 
         LocalAdmin.Configuration.LaShowStdoutStderr = BoolInput("Should standard outputs (contain a lot of debug information) be visible on the LocalAdmin live view?");
         LocalAdmin.Configuration.LaNoSetCursor = BoolInput("Should cursor position management be DISABLED (disable only if you are experiencing issues with the console, may cause issues especially on linux)?");
-        LocalAdmin.Configuration.EnableTrueColor = BoolInput("Should True Color output be enabled (disable if you can see some weird characters in some output lines, recommended to DISABLE if running LocalAdmin on WINDOWS)?");
+        LocalAdmin.Configuration.EnableTrueColor = BoolInput("Should True Color output be enabled (disable if you can see some weird characters in some output lines, recommended to DISABLE if running LocalAdmin on Windows)?");
```

Comment:
Invalid change. "WINDOWS" was intentionally typed in uppercase just like the "DISABLE" a couple of words before that.